Automattic/woocommerce-payments
# Fix error log messages incorrecty categorized as info

Fixes #9889 

#### Changes proposed in this Pull Request

Calls to `Logger::log_error()` were being incorrectly labelled as `info`

This PR passes the log level categorization onto the WooCommerce logger.

#### Testing instructions

* In WooPayments Dev Tools, select `Force the WCPay plugin to act as disconnected from the WCPay Server` and save settings.
* Visit the WooPayments dashboard
* View the logs, ensure you see `error` level logs.

-------------------


**Post merge**
